src/Compilers/CSharp/Portable/Binder/Binder_Invocation.cs

```diff
@@ -471,11 +471,6 @@ private ImmutableArray<BoundExpression> BuildArgumentsForDynamicInvocation(Analy
                         // error CS1978: Cannot use an expression of type '__arglist' as an argument to a dynamically dispatched operation
                         Error(diagnostics, ErrorCode.ERR_BadDynamicMethodArg, arg.Syntax, "__arglist");
                     }
-                    else if (arg.IsLiteralDefault())
```

Comment:
does this mean that 'default' is now a valid argument to a dynamic method after this PR?

Reply:
No, it's still an error. The errors that I removed are now covered by a common error which comes when we force expressions to get a type (the error is: no target type for `default`).

---
In reply to: [313659821](https://github.com/dotnet/roslyn/pull/37596#discussion_r313659821) [](ancestors = 313659821)